### PR TITLE
linuxKernel.kernels.linux_5_19: drop

### PR DESCRIPTION
**Pull request by lovesegfault**
The 5.19.x series has reached EOL, and is no longer supported upstream.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
makes sense to me, should probably remove it before 22.11 branch off

cc @NixOS/nixos-release-managers

File: pkgs/top-level/linux-kernels.nix [hunk 1]
```diff
@@ -193,7 +188,7 @@ in {
        else testing;
 
     linux_testing_bcachefs = callPackage ../os-specific/linux/kernel/linux-testing-bcachefs.nix rec {
-      kernel = linux_5_19;
+      kernel = linux_6_0;
```
Comment (jonringer) [inline pkgs/top-level/linux-kernels.nix:191]: not sure if this is the right move. Seems like most a lot of broken builds:
```
https://github.com/NixOS/nixpkgs/pull/198992

54 packages marked as broken and skipped:
linuxKernel.packages.linux_testing_bcachefs.akvcam linuxKernel.packages.linux_testing_bcachefs.ax99100 linuxKernel.packages.linux_testing_bcachefs.can-isotp linuxKernel.packages.linux_testing_bcachefs.ena linuxKernel.packages.linux_testing_bcachefs.intel-speed-select linuxKernel.packages.linux_testing_bcachefs.ixgbevf linuxKernel.packages.linux_testing_bcachefs.kvdo linuxKernel.packages.linux_testing_bcachefs.mwprocapture linuxKernel.packages.linux_testing_bcachefs.mxu11x0 linuxKernel.packages.linux_testing_bcachefs.ndiswrapper linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy340 linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy390 linuxKernel.packages.linux_testing_bcachefs.nvidiabl linuxKernel.packages.linux_testing_bcachefs.openafs linuxKernel.packages.linux_testing_bcachefs.openafs_1_8 linuxKernel.packages.linux_testing_bcachefs.phc-intel linuxKernel.packages.linux_testing_bcachefs.r8125 linuxKernel.packages.linux_testing_bcachefs.r8168 linuxKernel.packages.linux_testing_bcachefs.rtl8188eus-aircrack linuxKernel.packages.linux_testing_bcachefs.rtl8192eu linuxKernel.packages.linux_testing_bcachefs.rtl8723bs linuxKernel.packages.linux_testing_bcachefs.rtl88xxau-aircrack linuxKernel.packages.linux_testing_bcachefs.sch_cake linuxKernel.packages.linux_testing_bcachefs.tbs linuxKernel.packages.linux_testing_bcachefs.vmware linuxKernel.packages.linux_testing_bcachefs.zfs linuxKernel.packages.linux_testing_bcachefs.zfsStable linuxPackages_testing_bcachefs.akvcam linuxPackages_testing_bcachefs.ax99100 linuxPackages_testing_bcachefs.can-isotp linuxPackages_testing_bcachefs.ena linuxPackages_testing_bcachefs.intel-speed-select linuxPackages_testing_bcachefs.ixgbevf linuxPackages_testing_bcachefs.kvdo linuxPackages_testing_bcachefs.mwprocapture linuxPackages_testing_bcachefs.mxu11x0 linuxPackages_testing_bcachefs.ndiswrapper linuxPackages_testing_bcachefs.nvidia_x11_legacy340 linuxPackages_testing_bcachefs.nvidia_x11_legacy390 linuxPackages_testing_bcachefs.nvidiabl linuxPackages_testing_bcachefs.openafs linuxPackages_testing_bcachefs.openafs_1_8 linuxPackages_testing_bcachefs.phc-intel linuxPackages_testing_bcachefs.r8125 linuxPackages_testing_bcachefs.r8168 linuxPackages_testing_bcachefs.rtl8188eus-aircrack linuxPackages_testing_bcachefs.rtl8192eu linuxPackages_testing_bcachefs.rtl8723bs linuxPackages_testing_bcachefs.rtl88xxau-aircrack linuxPackages_testing_bcachefs.sch_cake linuxPackages_testing_bcachefs.tbs linuxPackages_testing_bcachefs.vmware linuxPackages_testing_bcachefs.zfs linuxPackages_testing_bcachefs.zfsStable

73 packages failed to build:
linux_testing_bcachefs linuxPackages_testing_bcachefs.acpi_call linuxPackages_testing_bcachefs.apfs linuxPackages_testing_bcachefs.asus-ec-sensors linuxPackages_testing_bcachefs.asus-wmi-sensors linuxPackages_testing_bcachefs.batman_adv linuxPackages_testing_bcachefs.bbswitch linuxPackages_testing_bcachefs.broadcom_sta linuxPackages_testing_bcachefs.chipsec linuxPackages_testing_bcachefs.cryptodev linuxPackages_testing_bcachefs.ddcci-driver linuxPackages_testing_bcachefs.dddvb linuxPackages_testing_bcachefs.digimend linuxPackages_testing_bcachefs.dpdk linuxPackages_testing_bcachefs.dpdk-kmods linuxPackages_testing_bcachefs.evdi linuxPackages_testing_bcachefs.facetimehd linuxPackages_testing_bcachefs.fwts-efi-runtime linuxPackages_testing_bcachefs.gcadapter-oc-kmod linuxPackages_testing_bcachefs.hid-ite8291r3 linuxPackages_testing_bcachefs.hid-nintendo linuxPackages_testing_bcachefs.isgx linuxPackages_testing_bcachefs.it87 linuxPackages_testing_bcachefs.ithc linuxPackages_testing_bcachefs.jool linuxPackages_testing_bcachefs.kvmfr linuxPackages_testing_bcachefs.liquidtux linuxPackages_testing_bcachefs.lttng-modules linuxPackages_testing_bcachefs.mba6x_bl linuxPackages_testing_bcachefs.mbp2018-bridge-drv linuxPackages_testing_bcachefs.netatop linuxPackages_testing_bcachefs.new-lg4ff linuxPackages_testing_bcachefs.nvidia_x11 linuxPackages_testing_bcachefs.nvidia_x11_beta_open linuxPackages_testing_bcachefs.nvidia_x11_legacy470 linuxPackages_testing_bcachefs.nvidia_x11_production linuxPackages_testing_bcachefs.nvidia_x11_production_open linuxPackages_testing_bcachefs.nvidia_x11_vulkan_beta linuxPackages_testing_bcachefs.nvidia_x11_vulkan_beta_open linuxPackages_testing_bcachefs.openrazer linuxPackages_testing_bcachefs.prl-tools linuxPackages_testing_bcachefs.qc71_laptop linuxPackages_testing_bcachefs.rr-zen_workaround linuxPackages_testing_bcachefs.rtl8189es linuxPackages_testing_bcachefs.rtl8189fs linuxPackages_testing_bcachefs.rtl8812au linuxPackages_testing_bcachefs.rtl8814au linuxPackages_testing_bcachefs.rtl8821au linuxPackages_testing_bcachefs.rtl8821ce linuxPackages_testing_bcachefs.rtl8821cu linuxPackages_testing_bcachefs.rtl88x2bu linuxPackages_testing_bcachefs.rtw88 linuxPackages_testing_bcachefs.sysdig linuxPackages_testing_bcachefs.system76 linuxPackages_testing_bcachefs.system76-acpi linuxPackages_testing_bcachefs.system76-io linuxPackages_testing_bcachefs.systemtap linuxPackages_testing_bcachefs.tp_smapi linuxPackages_testing_bcachefs.tuxedo-keyboard linuxPackages_testing_bcachefs.usbip linuxPackages_testing_bcachefs.v4l2loopback linuxPackages_testing_bcachefs.v86d linuxPackages_testing_bcachefs.veikk-linux-driver linuxPackages_testing_bcachefs.vendor-reset linuxPackages_testing_bcachefs.vhba linuxPackages_testing_bcachefs.virtio_vmmci linuxPackages_testing_bcachefs.virtualbox linuxPackages_testing_bcachefs.virtualboxGuestAdditions linuxPackages_testing_bcachefs.vmm_clock linuxPackages_testing_bcachefs.xone linuxPackages_testing_bcachefs.xpadneo linuxPackages_testing_bcachefs.zenpower linuxPackages_testing_bcachefs.zfsUnstable

7 packages built:
linuxPackages_testing_bcachefs.cpupower linuxPackages_testing_bcachefs.hyperv-daemons linuxPackages_testing_bcachefs.perf linuxPackages_testing_bcachefs.ply linuxPackages_testing_bcachefs.tmon linuxPackages_testing_bcachefs.turbostat linuxPackages_testing_bcachefs.x86_energy_perf_policy
```

**Review by jonringer — DISMISSED** [review]

File: pkgs/top-level/linux-kernels.nix (hunk 1 above)
Comment (lovesegfault) [reply pkgs/top-level/linux-kernels.nix:191]: cc. @davidak @Madouura

**Comment by lovesegfault** [comment]
Result of `nixpkgs-review pr 198992` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>25 packages marked as broken and skipped:</summary>
  <ul>
    <li>linuxKernel.packages.linux_testing_bcachefs.akvcam</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ax99100</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.can-isotp</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ena</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.intel-speed-select</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ixgbevf</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.kvdo</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.mwprocapture</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.mxu11x0</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ndiswrapper</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy340</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy390</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidiabl</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.openafs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.phc-intel</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.r8125</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.r8168</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8188eus-aircrack</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8192eu</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8723bs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl88xxau-aircrack</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.sch_cake</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.tbs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.vmware</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.zfs</li>
  </ul>
</details>
<details>
  <summary>10 packages failed to build:</summary>
  <ul>
    <li>linuxKernel.packages.linux_testing_bcachefs.apfs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.evdi</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.lttng-modules</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy470</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_vulkan_beta</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_vulkan_beta_open</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.prl-tools</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8814au</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8821ce</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtlwifi_new</li>
  </ul>
</details>
<details>
  <summary>70 packages built:</summary>
  <ul>
    <li>linuxKernel.kernels.linux_testing_bcachefs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.acpi_call</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.asus-ec-sensors</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.asus-wmi-sensors</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.batman_adv</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.bbswitch</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.broadcom_sta</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.chipsec</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.cpupower</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.cryptodev</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ddcci-driver</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.dddvb</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.digimend</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.dpdk</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.dpdk-kmods</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.facetimehd</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.fwts-efi-runtime</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.gcadapter-oc-kmod</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.hid-ite8291r3</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.hid-nintendo</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.hyperv-daemons</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.isgx</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.it87</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ithc</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.jool</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.kvmfr</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.liquidtux</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.mba6x_bl</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.mbp2018-bridge-drv</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.netatop</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.new-lg4ff</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_production</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_production_open</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.openrazer</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.perf</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.ply</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.qc71_laptop</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rr-zen_workaround</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8189es</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8189fs</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8812au</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8821au</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl8821cu</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.rtl88x2bu</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.sysdig</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.system76</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.system76-acpi</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.system76-io</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.systemtap</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.tmon</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.tp_smapi</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.turbostat</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.tuxedo-keyboard</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.usbip</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.v4l2loopback</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.v86d</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.veikk-linux-driver</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.vendor-reset</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.vhba</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.virtio_vmmci</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.virtualbox</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.virtualboxGuestAdditions</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.vmm_clock</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.x86_energy_perf_policy</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.xone</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.xpadneo</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.zenpower</li>
    <li>linuxKernel.packages.linux_testing_bcachefs.zfsUnstable</li>
  </ul>
</details>

File: pkgs/top-level/linux-kernels.nix (hunk 1 above)
Comment (lovesegfault) [reply pkgs/top-level/linux-kernels.nix:191]: Fixed by updating the patches

**Pull request merged**